src/Components/Navigation.js

```diff
@@ -1,24 +1,26 @@
 import { NavLink } from 'react-router-dom';
+// import { FaUserAlt } from 'react-icons/fa';
```

Comment:
- [ ] Please don't push commented code it's your codebase that makes it messy if you want to check your previous code then check your GitHub commit history.